execute/executetest/transformation.go

```diff
@@ -1,15 +1,49 @@
 package executetest
 
 import (
+	"math"
 	"sort"
 	"testing"
 
+	"github.com/gonum/floats"
+
 	"github.com/google/go-cmp/cmp"
 	"github.com/google/go-cmp/cmp/cmpopts"
+
 	"github.com/influxdata/flux"
 	"github.com/influxdata/flux/execute"
 )
 
+// Two floating point values are considered
+// equal if they are within tol of each other.
+const tol float64 = 1e-25
+
+// The maximum number of floating point values that are allowed
+// to lie between two float64s and still be considered equal.
+const ulp uint = 2
+
+// Comparison options for floating point values.
+// NaNs are considered equal, and float64s must
+// be sufficiently close to be considered equal.
+var floatOptions = cmp.Options{
+	cmpopts.EquateNaNs(),
+	cmp.FilterValues(func(x, y float64) bool {
+		return !math.IsNaN(x) && !math.IsNaN(y)
```

Comment:
What does this check do?

Reply:
It only executes the following comparer function when neither float is a NaN.